lona/html/parsing.py

```diff
@@ -143,7 +143,7 @@ def handle_starttag(self, tag, attrs, self_closing=False):
         node_kwargs['self_closing_tag'] = self_closing
 
         # setup node
-        for key in ('id', 'class', 'style'):
+        for key in ('id', 'class', 'style', 'value'):
```

Comment:
@maratori: That means:

```python
select = Select(
    Option('Foo', value='foo', selected=True),
    multiple=False,
)

option = Option('Bar', value='bar', selected=True)

select.add_option(option)

assert option.selected == False
```

Right?

I agree that this behavior would be consistent with the API i proposed, but maybe that's a little bit to much magic because  what should we do in cases like these:

```python
select = Select(
    Option('Foo', value='foo', selected=True),
    Option('Bar', value='bar', selected=True),
)

select.multiple = True

assert select.options[0].selected == True
assert select.options[1].selected == True  # fails
```

In the browser that would work. Do we want this behavior? Or do we want `Option.selected` to be dumb and the "clever" metrics like "return the first selected option if multiple are selected and it's no multi-select" are in `Select.value`, `Select.selected_values` ?

Reply:
> Right?

This seems to be not related to my original comment.

To answer your great question, I need to take some time to play around with this API and with browser.

Reply:
> This seems to be not related to my original comment.

No? If i read your example correctly, you set two options to selected in the HTML string but assert only one `selected == True` (b and c). Am i missing something?

Reply:
I haven't run test, but from reading the code I think parsing string HTML doesn't work correctly with new select API. That's because `Select.append` behave differently from `Select.add_option`.

Reply:
Oh! You are right. Maybe a second reason to strip the proposed `Option.selected` API